### DISCUSSION TITLE
Mi página de la pastelería

### DISCUSSION THREAD

**Discussion by GloriaCoralCerecedo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#280] [Show and tell]
Hola, buenas noches.

Acabo de subir mi práctica, ya que la subí en github pages entonces está en línea. No es difícil.

Yo ya había tenido conocimiento de HTML, css y Js pero no tanto. 

Igual intente hacer todo sin boostrap, ni js. Solo utilice HTML y CSS.

También de que sea  responsiva aunque sinceramente en el cel se ve fatal jaja solo en la PC se ve bien.  

Bueno esta es mi página 

https://gloriacoralcerecedo.github.io/IntroHTML/

**Comment by RodolfoMorquecho** [comment]
Te quedo genial Gloria, por cierto tengo la duda de como subiste la pag al repo, yo subi la carpeta donde esta todo el codigo e imagenes pero debo de subir el link de la pag? disculpa la ignorancía.

**Reply by GloriaCoralCerecedo** [reply]
Al repositorio exactamente subes los archivos y imágenes. 

Tienes que enviar el link de tu repositorio

Y si está en línea subes el link, si no.

**Reply by JuanSalvador73** [reply]
En tu repositorio te da la opción de "add files" que es donde subes tus archivos. Si tu sitio está publicado en algún servidor gratis - o de paga - simplemente copias el url y lo pegas en el formulario de entrega. También puedes señalar ese mismo url en tu repositorio, en la parte de edición.

**Comment by JuanSalvador73** [comment]
Muy buen sitio, interesante como manejaste la secuencia de páginas.

**Reply by GloriaCoralCerecedo** [reply]
Gracias 😁

**Comment by OscarPedroza** [comment]
Que tal compañera, gran página y muy completa en cuanto sabores y decoraciones.

**Reply by GloriaCoralCerecedo** [reply]
Muchas gracias 😁
